src/matchers/toEqualText/index.test.ts

```diff
@@ -10,71 +11,72 @@ describe("toEqualText", () => {
     it("positive frame", async () => {
       await page.setContent(`<iframe src="https://example.com"></iframe>`)
       const iframe = await page.$("iframe")
-      const result = await toEqualText(iframe!, "h1", "Example Domain")
-      expect(result.pass).toBe(true)
-      expect(result.message()).toMatchSnapshot()
+      await expect(iframe!).toEqualText("h1", "Example Domain")
```

Comment:
I updated the tests to use the matcher directly in the test rather than testing the results of it.  The reason for this is that the matcher is now using more of `jest.MatcherContext` such as `printExpected`, `matcherHint`, etc.